task.py

```diff
@@ -383,6 +384,22 @@ def get_pod_ip(self) -> str:
             raise RuntimeError("Failure to get static.podIP for {self.pod_name}")
         return pod_ip
 
+    def get_secondary_ip(self) -> str:
+        if self.ts.connection.secondary_network_nad is None:
+            raise ValueError("Secondary Network is missing")
```

Comment:
this check can be dropped now. The value is always set (possibly to `f"{namespace}/ocp-secondary"`.